compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/DeclarationStubGenerator.kt

```diff
@@ -76,8 +76,10 @@ class DeclarationStubGenerator(
         if (symbol is IrFieldSymbol && (symbol.descriptor as? WrappedPropertyDescriptor)?.isBound() == true) {
             return generateStubBySymbol(symbol, symbol.descriptor)
         }
+        val signature = symbol.signature
+            ?: error("Symbol is not public API. Expected signature for symbol: ${symbol.descriptor}")
```

Comment:
I mean 
`findDescriptorBySignature(symbol.signature
            ?: error("Symbol is not public API. Expected signature for symbol: ${symbol.descriptor}")`